jobs/godojo/templates/bin/pre-start.erb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+
+cd /var/vcap/jobs/godojo/bin
+/var/vcap/jobs/godojo/packages/godojo/godojo
```

Comment:
What is this doing?

Reply:
This is running the godojo installation script for setting up DefectDojo. It is necessary to do it as a pre-start script so that it doesn't do the whole install process after a simple restart